flutter/lib/src/default_integrations.dart

```diff
@@ -494,3 +497,66 @@ class LoadReleaseIntegration extends Integration<SentryFlutterOptions> {
         .replaceAll('\u{0000}', '');
   }
 }
+
+/// Enrich [SentryTransaction] objects with native app data for mobile vitals.
+class MobileVitalsIntegration extends Integration<SentryFlutterOptions> {
+  MobileVitalsIntegration(this._nativeWrapper);
+
+  final SentryNativeWrapper _nativeWrapper;
+
+  @override
+  FutureOr<void> call(Hub hub, SentryFlutterOptions options) {
+    if (options.autoAppStartFinish) {
+      SchedulerBinding.instance?.addPostFrameCallback((timeStamp) {
+        options.appStartFinish = DateTime.now();
+      });
```

Comment:
I'd rather get the `SchedulerBinding` via ctor with default value of `SchedulerBinding.instance`.
The idea is also to log out if `SchedulerBinding` is null, so users know why the feature isn't working.
It can also be used for testing/mocking.

Reply:
Default value did not work as it is expecting `const` which didn't work here, but i moved it to the ctor so we can inject 👍